client/components/marketing-survey/cancel-purchase/index.jsx

```diff
@@ -0,0 +1,357 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import shuffle from 'lodash/shuffle';
+
+/**
+ * Internal Dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextarea from 'components/forms/form-textarea';
+
+const CancelPurchaseSurvey = React.createClass( {
```

Comment:
The naming is a little inconsistent, because your component name should match your filename (other than the camelCase of course). So if the filename is `cancel-purchase/index.jsx`, then by convention this component would be `const CancelPurchase = React.createClass( {`, and the css classNames should be in the form `.cancel-purchase__[subelement]`.

Because of this, all of your classNames are triggering lint warnings.

![screen shot 2016-07-26 at 3 15 42 pm](https://cloud.githubusercontent.com/assets/134044/17153802/ead85004-5343-11e6-9bb5-ee34c657a372.png)

Do you have linting set up in your code editor? You'll definitely want that.

`CancelPurchase` seems a little generic though, so I'd probably name this `CancelPurchaseForm` and move the file to `cancel-purchase-form/index.jsx` and change the classNames to `cancel-purchase-form__[subelement]`